identity/src/lib/rpc/server.rs

```diff
@@ -6,7 +6,9 @@ use diesel::prelude::*;
 use diesel::r2d2::{ConnectionManager, PooledConnection};
 use tarpc::context;
 
-use super::service::*;
+use helpers::rpc::{Error, RpcResult};
```

Comment:
You could also re-export these two from your models. (Am doing that in `book`)

Reply:
I will address this in a different PR. There are going to be some changes on the models anyway.